lms/djangoapps/lti_provider/users.py

```diff
@@ -53,9 +55,13 @@ def create_lti_user(lti_user_id, lti_consumer):
             edx_user = User.objects.create_user(
                 username=edx_user_id,
                 password=edx_password,
-                email='{}@lti.example.com'.format(edx_user_id)
+                email='{}@lti.example.com'.format(edx_user_id),
```

Comment:
It's mostly a paranoia thing. While we shouldn't ever be sending emails to these folks, if something does happen, I'd like ops to have the option to set it to point at a mail server within their control. Defaulting it to example.com is fine.